pkg/volume/nfs/nfs.go

```diff
@@ -237,24 +237,24 @@ func (c *nfsUnmounter) TearDown() error {
 func (c *nfsUnmounter) TearDownAt(dir string) error {
 	notMnt, err := c.mounter.IsLikelyNotMountPoint(dir)
 	if err != nil {
-		glog.Errorf("Error checking IsLikelyNotMountPoint: %v", err)
+		glog.Errorf("nfs: Error checking IsLikelyNotMountPoint: %v", err)
 		return err
 	}
 	if notMnt {
-		return os.Remove(dir)
+		return os.RemoveAll(dir)
 	}
 
 	if err := c.mounter.Unmount(dir); err != nil {
-		glog.Errorf("Unmounting failed: %v", err)
+		glog.Errorf("nfs: Unmounting failed: %v", err)
 		return err
 	}
 	notMnt, mntErr := c.mounter.IsLikelyNotMountPoint(dir)
 	if mntErr != nil {
-		glog.Errorf("IsLikelyNotMountPoint check failed: %v", mntErr)
+		glog.Errorf("nfs: IsLikelyNotMountPoint check failed: %v", mntErr)
 		return mntErr
 	}
 	if notMnt {
-		if err := os.Remove(dir); err != nil {
+		if err := os.RemoveAll(dir); err != nil {
```

Comment:
Curious, what motivated this change. After a volume is unmounted the directory should already be empty right?